lib/streamlit/app_session.py

```diff
@@ -48,6 +48,11 @@ class AppSessionState(Enum):
     SHUTDOWN_REQUESTED = "SHUTDOWN_REQUESTED"
 
 
+def _generate_scriptrun_id() -> str:
+    """Randomly generate a unique ID for a script execution."""
+    return str(uuid.uuid4())
+
+
```

Comment:
The previous implementation did a base58.encode/decode to convert the uuid to a string, but I don't see any reason not to just use the `str()` function.